src/os/bluestore/BlueStore.cc

```diff
@@ -8740,6 +8742,8 @@ void BlueStore::_txc_finish(TransContext *txc)
       dout(10) << __func__ << " empty zombie osr " << osr << " already reaped" << dendl;
     }
   }
+  logger->set(l_bluestore_fragmentation,
+    (uint64_t)alloc->get_fragmentation(min_alloc_size));
```

Comment:
this should be *1000 or something since it's a value between 0 and 1, right?

Reply:
yep